rest-server/engines/src/main/scala/com/actionml/engines/ur/URDataset.scala

```diff
@@ -32,11 +33,12 @@ import scala.language.reflectiveCalls
   *
   * @param engineId The Engine ID
   */
-class URDataset(engineId: String) extends Dataset[GenericEvent] with JsonParser {
-
+class URDataset(val engineId: String) extends Dataset[GenericEvent] with JsonParser {
+  override lazy val dbName: String = engineId
```

Comment:
no, error. The dbname is not always the engineId, you MUST get it from the Store or make the DAO know what store it uses and get it from the DAO.

This is an error since sometimes the dbname is NOT the enginId.